common-o2.sh

```diff
@@ -0,0 +1,48 @@
+package: Common-O2
+version: "%(tag_basename)s"
+tag: v0.2.0
+requires:
+  - boost
+  - "GCC-Toolchain:(?!osx)"
+build_requires:
+  - CMake
+source: https://github.com/AliceO2Group/Common
+incremental_recipe: |
+  make ${JOBS:+-j$JOBS} install
+  mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+---
+#!/bin/sh
+
+case $ARCHITECTURE in
+  osx*) BOOST_ROOT=$(brew --prefix boost) ;;
+esac
+
+cmake $SOURCEDIR                                              \
+      -DCMAKE_INSTALL_PREFIX=$INSTALLROOT                     \
+      -DCMAKE_PREFIX_PATH=$PROTOBUF_ROOT                      \
+      ${BOOST_ROOT:+-DBOOST_ROOT=$BOOST_ROOT}                 \
```

Comment:
Do we really need `-DBOOST_ROOT` plus `-DBoost_DIR` and `-DBoost_INCLUDE_DIR`? Normally the second is sufficient. Also, the check should be on `BOOST_VERSION` and not `BOOST_ROOT`.